src/phylum/constants.py

```diff
@@ -58,6 +56,8 @@
     "*.spdx.yaml": "spdx",
     "*.spdx.yml": "spdx",
     "*.spdx": "spdx",
+    "*bom.json": "cyclonedx",
+    "*bom.xml": "cyclonedx",
```

Comment:
Assuming this also matches just `bom.json` and `bom.xml`.

Reply:
Yes. It was confirmed with local testing by adding 4 files to the `phylum-ci` project directory and running the code here:

`bom.json`, `bom.xml`, `project.bom.json`, `project.bom.xml`

```
---TRIMMED---
No valid lockfiles were provided as arguments. Checking the `.phylum_project` file ...
No valid lockfiles in the `.phylum_project` file. An attempt will be made to detect lockfiles.
Detected lockfile(s): [PosixPath('/Users/maxrake/dev/phylum/phylum-ci/poetry.lock'), PosixPath('/Users/maxrake/dev/phylum/phylum-ci/bom.json'),
PosixPath('/Users/maxrake/dev/phylum/phylum-ci/project.bom.json'), PosixPath('/Users/maxrake/dev/phylum/phylum-ci/project.bom.xml'),
PosixPath('/Users/maxrake/dev/phylum/phylum-ci/bom.xml')]
Valid detected lockfiles: [bom.json, bom.xml, poetry.lock, project.bom.json, project.bom.xml]
Lockfiles in use: [bom.json, bom.xml, poetry.lock, project.bom.json, project.bom.xml]
---TRIMMED---
```